src/base.rs

```diff
@@ -73,8 +73,17 @@ extern "C" {
     pub static kCFAllocatorNull: CFAllocatorRef;
     pub static kCFAllocatorUseContext: CFAllocatorRef;
 
-    pub fn CFGetTypeID(cf: CFTypeRef) -> CFTypeID;
-
+    pub fn CFGetAllocator(cf: CFTypeRef) -> CFAllocatorRef;
+    pub fn CFGetRetainCount(cf: CFTypeRef) -> CFIndex;
+    pub fn CFMakeCollectible(cf: CFTypeRef) -> CFTypeRef;
     pub fn CFRetain(cf: CFTypeRef) -> CFTypeRef;
     pub fn CFRelease(cf: CFTypeRef);
+
+    pub fn CFEqual(cf1: CFTypeRef, cf2: CFTypeRef) -> Boolean;
+    pub fn CFHash(cf: CFTypeRef) -> CFHashCode;
+
+    pub fn CFCopyDescription(cf: CFTypeRef) -> CFStringRef;
+    pub fn CFCopyTypeIDDescription(cf: CFTypeRef) -> CFStringRef;
+    pub fn CFGetTypeID(cf: CFTypeRef) -> CFTypeID;
+    pub fn CFShow(obj: CFTypeRef);
```

Comment:
Looks like it's defined in `CFString.h`. Do you mind moving it to `src/string.rs`?